### PR TITLE
Adds Export exception list API docs

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Contributes to #3061 by documenting the exceptions list export API.

This PR documents the parameters that exist in all versions. Once this PR is merged, I'll open a separate one to add the `include_expired_exceptions` parameter, which exists only in versions 8.7 and newer.

Preview: [Export exception list](https://security-docs_4316.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-export-exception-list.html)

### CODE REVIEW

File: docs/detections/api/exceptions/api-export-exception-list.asciidoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+`POST <kibana host>:<port>/api/exception_lists/_export`
+
+===== URL query parameters
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`id` |String |ID of the exception list. |No.
+|`list_id` |String |List ID of the exception list. |No.
```
Comment (dplumlee) [inline docs/detections/api/exceptions/api-export-exception-list.asciidoc:17]: Both of these fields are required going back to when the route was created

**Review by dplumlee — COMMENTED** [review]
I think this looks good for the fields going back all the way to `8.0` aside from the required field changes. @yctercero you have a bit more experience in this area if you wanted to review as well

**Comment by dplumlee** [comment]
Also I think this goes back to `7.11`, is there a separate process for adding this to `7.11` and `7.12` too? Are we still adding things to `7.x` docs?

File: docs/detections/api/exceptions/api-export-exception-list.asciidoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+|`namespace_type` |String a|Determines whether the exported exception list is associated with a single {kib} space or available in all spaces, where
```
Comment (yctercero) [inline docs/detections/api/exceptions/api-export-exception-list.asciidoc:18]: This one is also required.

Comment (yctercero) [reply docs/detections/api/exceptions/api-export-exception-list.asciidoc:18]: Ignore! I'm wrong - doc is correct that it defaults to `single`

**Comment by natasha-moore-elastic** [comment]
> Also I think this goes back to `7.11`, is there a separate process for adding this to `7.11` and `7.12` too? Are we still adding things to `7.x` docs?

That's a good question – I think we generally backport as far as `8.0` (and maybe `7.17` as the last `7.x` version), but not beyond that. @elastic/security-docs correct me if I'm wrong?

File: docs/detections/api/exceptions/api-export-exception-list.asciidoc (hunk 1 above)
Comment (natasha-moore-elastic) [reply docs/detections/api/exceptions/api-export-exception-list.asciidoc]: Resolved in [a56ddea](https://github.com/elastic/security-docs/pull/4316/commits/a56ddea5156c24f208b70a5d38570833073d288d).

**Review by yctercero — APPROVED** [review]
LGTM - thanks!

**Review by dplumlee — APPROVED** [review]

**Review by joepeeples — COMMENTED** [review]
One li'l suggestion, thanks!

**Review by joepeeples — APPROVED** [review]

**Comment by natasha-moore-elastic** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|7.x|The branch "7.x" does not exist|
|✅|7.13|[](https://github.com/elastic/security-docs/pull/4346)|
|✅|7.12|[](https://github.com/elastic/security-docs/pull/4347)|
|✅|7.11|[](https://github.com/elastic/security-docs/pull/4348)|

### Manual backport
To create the backport manually run:
```
backport --pr 4316
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**